pkg/3scale/amp/auto-generated-templates/amp/amp-eval-s3.yml

```
@@ -4499,9 +4499,7 @@ parameters:
   name: AWS_BUCKET
 - description: AWS Region to use in S3 Storage for assets.
   name: AWS_REGION
-- description: Storage Service URL scheme to use for the S3 API-compatible storage
-    service
-  name: STORAGE_SERVICE_ENDPOINT_SCHEME
-- description: Storage Service URL host and optionally a port separated by a colon
-    (:) to use as the S3 API-compatible storage service
-  name: STORAGE_SERVICE_ENDPOINT_HOST
+- description: AWS S3 provider endpoint protocol. HTTP or HTTPS.
```

```suggestion
- description: AWS S3 compatible provider endpoint protocol. HTTP or HTTPS.
```

not sure if we should add something like that to all descriptions to make clear it's not just AWS?

---

I will update with your change. 

That file is automatically generated, so should not be edited manually.